tests/test_free_energy.py

```diff
@@ -42,7 +44,7 @@ def test_absolute_free_energy():
 
     lambda_schedule = np.linspace(0, 1.0, 4)
     equil_steps = 1000
-    prod_steps = 1000
+    prod_steps = 5000
```

Comment:
These values increased as we were previously getting more samples for TI than we do for BAR. Could change the estimator code to take in a flag to determine how many frames to save.

Reply:
Does this impact the time to run the test?

Reply:
It does, increases both tests that had this change by about 20 seconds.

Reply:
Fixed this by parameterizing it in [e19ead8](https://github.com/proteneer/timemachine/pull/700/commits/e19ead869f7eb48f47401759356432de3b5485c0)